shared/editor/plugins/Prism.ts

```diff
@@ -7,6 +7,7 @@ import refractor from "refractor/core";
 
 export const LANGUAGES = {
   none: "None", // additional entry to disable highlighting
+  mermaidjs: "Mermaid Diagrams",
```

Comment:
```suggestion
  mermaidjs: "Mermaid Diagram",
```

Prefer singular

Reply:
Note it would probably be relatively easy as a followup to create a /mermaid slash command that creates a new code block with the language pre-selected.